Tasks/SqlDacpacDeploymentOnMachineGroup/ExecuteScriptsInTransaction.sql

```diff
@@ -0,0 +1,282 @@
+PRINT 'Begin Sql scripts execution'
+SET XACT_ABORT ON
+SET NOCOUNT ON
+
+DECLARE @_acquireLock VARCHAR(64)
+DECLARE @_acquireLockMilliseconds INT
+DECLARE @_acquireLockMaxAttempts INT
+DECLARE @_longRunningThresholdMilliSeconds INT
+DECLARE @_lastAttemptsToKillSessions INT
+DECLARE @_isHosted INT
+
+PRINT 'Adding SQL files into temporary table _tmpSqlFilesTable'
+
+IF OBJECT_ID (N'_tmpSqlFilesTable', N'U') IS NOT NULL 
+BEGIN
+   PRINT 'Temporary table exists already. Dropping table _tmpSqlFilesTable'
+   DROP TABLE _tmpSqlFilesTable
+END
+CREATE TABLE _tmpSqlFilesTable (ScriptIndex INT NOT NULL IDENTITY, SqlScriptContent NVARCHAR(MAX) NULL)
```

Comment:
>SqlScriptContent NVARCHAR(MAX) NULL [](start = 67, length = 35)

1. The only way we are inserting into this table is via View.
2. This column allows NULL

See the statement from this artitcle [http://jagbarcelo.blogspot.in/2006/09/identity-columns-in-view-treated-as.html](http://jagbarcelo.blogspot.in/2006/09/identity-columns-in-view-treated-as.html)
_INSERT statements must specify values for any columns in the underlying table that do not allow null values and have no DEFAULT definitions_

Should we remove NULL from the table definition?
Or should we have a deafult?